hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/MaintainerEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package pl.allegro.tech.hermes.api.endpoints;
+
+import pl.allegro.tech.hermes.api.Maintainer;
+
+import javax.ws.rs.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- javax\.ws\.rs\.\*\.